prototype/app/views/get-involved.html

```diff
@@ -0,0 +1,42 @@
+{% extends "layouts/main.html" %}
+
+{% set pageTitle = "Get involved" %}
+
+{% block content %}
+
+<div class="govuk-width-container ">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+      <h1 class="govuk-heading-l">Get involved</h1>
+      <p>As the Consultation analyser is in the test and learn phase we need to learn how it compares to other methods you've used. How you can get involved depends on whether the consultation has already been analysed.</p>
+
+      <h2 class="govuk-body-l govuk-!-padding-top-5">You have recently <strong>completed</strong> a consultation</h2>
+      <p>If you have recently completed an analysed a consultation the steps are:</p>
+      <ol class="govuk-list govuk-list--number">
+        <li>Agree <strong>data sharing</strong>. See draft <a href="https://docs.google.com/document/d/1uh4mVuQBEBfp_oSfzyIbsPzR3owS9PNK/edit?usp=sharing&ouid=103333141005222978195&rtpof=true&sd=true">Memorandum of Understanding</a>.</li>
+        <li>We <strong>survey and interview</strong> the team involved in analysing the consultation.</li>
+        <li>You get <strong>access to the Consultation analyser</strong> to share your data and see the themes it creates.</li>
+        <li>We <strong>survey and interview</strong> those involved to understand your experience.</li>
+        <li>We <strong>compare the themes</strong> generated by the original analysis and the Consultation analyser.</li>
+        <li>Final <strong>debrief</strong> session to discuss the findings.</li>
+      </ol>
+
+      <h2 class="govuk-body-l govuk-!-padding-top-5">You have an <strong>upcoming</strong> consultation and you have other analysis already planned</h2>
+      <p>If you have an upcoming consultation and you will be running the analysis in-house, and would like to use Consultation Analyser alongside the steps are:</p>
```

Comment:
Inconsistent "Consultation Analyser" (but was wrong in doc!)

Reply:
Thank you! Fixed now.